server/authorization.go

```diff
@@ -214,47 +259,109 @@ func (a *AuthorizationServer) Check(ctx gocontext.Context, req *authv3.CheckRequ
 	authContext, err := a.handler.authMan.Authenticate(rootContext, apiKey, claims, a.handler.apiKeyClaim)
 	switch err {
 	case auth.ErrNoAuth:
-		return a.unauthenticated(req, envRequest, tracker, api), nil
+		return a.handleFault(
```

Comment:
line breaking for arguments is not recommended in go. Please convert it to the one-liner even if it's very long.

Reply:
Let me know if I am wrong though ;-)

Reply:
Thanks for pointing out. 
From what I can find (refer: https://stackoverflow.com/a/34848928) my use of line-breaks works just fine and it makes code much more readable.

Reply:
@rockspore is correct here; prefer to not break lines unless it is absolutely required. Our style prefers refactoring to reduce line length, if required, over breaking lines solely for readability.